packages/@o3r/eslint-config-otter/schematics/ng-update/index.ts

```diff
@@ -8,6 +8,9 @@ import {
 import {
   addStylistic,
 } from './v10.0/stylistic';
+import {
+  updateEslintRecommended,
+} from './v11.5/update-configs/update-configs';
```

Comment:
```suggestion
} from './v11.6/update-configs/update-configs';
```
folder should be renamed